.github/workflows/report_refresh.yml

```diff
@@ -0,0 +1,18 @@
+# Update the nightly report stats etc.
+
+name: Report refresh
+on:
+  schedule:
+    - cron: '0 14 * * *'
```

Comment:
We might want to merge first the manual one and only add the periodic one once that's tested.